compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -141,12 +141,24 @@ class ReplDriver(settings: Array[String],
     }
   }
 
+  final def runWrapper(initialState: State = initState): State = {
```

Comment:
I would not expose this as public API. I would move it into `runUntilQuit`.

Reply:
I modified `runUntilQuit` now. Please have a look